ehrql/query_model/nodes.py

```diff
@@ -245,11 +245,13 @@ class Parameter(OneRowPerPatientSeries[T]):
 class SelectTable(ManyRowsPerPatientFrame):
     name: str
     schema: TableSchema
+    required_permission: str | None = None
```

Comment:
Is it safer for the default permission to be some placeholder value rather than `None`, so that all tables have to override it? Or is this something we'll definitely consider when we add a new table?